build-image-src/Dockerfile-provided

```diff
@@ -48,6 +48,12 @@ ENV PATH=$PATH:/usr/local/opt/lambda-builders/bin
 # Python for it to be picked up during `sam build`
 RUN pip3 install wheel
 
+# Install Go
+RUN curl -L https://go.dev/dl/go1.19.linux-amd64.tar.gz | tar -zx -C /usr/local
```

Comment:
@hawflau 
Should we install latest Go for this image as well? (just like we did for the other provided runtimes)

Reply:
my miss